### PR TITLE
ts

### PR DESCRIPTION
**Pull request by tovade**

### CODE REVIEW

**Comment by AngeloCore** [comment]
OwO

File: struct/interfaces.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+export interface DefaultOptions {
```
Comment (Koolwiza) [inline struct/interfaces.ts:1]:
```suggestion
export const DefaultOptions {
```

File: struct/GiveawayManager.ts [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+  autoFetch: true,
+  fetchAll: true,
+});
+const messages = new Enmap({
+  name: "messages",
+  fetchAll: true,
+  autoFetch: true,
+});
+
+export { giveaways };
```
Comment (Koolwiza) [inline struct/GiveawayManager.ts:26]:
```suggestion
export { giveaways, messages };
```
Just for later, incase people want to make a command to check how many messages they have

File: struct/interfaces.ts (hunk 1 above)
Comment (tovade) [reply struct/interfaces.ts:1]: Interface is defining the type

Comment (Koolwiza) [reply struct/interfaces.ts:1]: Yeah but... Isn't that an object

Comment (tovade) [reply struct/interfaces.ts:1]: const is just an object interface is defining the type and if you look in the manager it is used as an type

Comment (Koolwiza) [reply struct/interfaces.ts:1]: Ah alright

**Comment by Koolwiza** [comment]
Can you add the utilization of the Manager?

**Comment by Koolwiza** [comment]
Like ending/rerolling

**Comment by tovade** [comment]
what?

**Pull request closed without merging**